scripts/imgtool.py

```diff
@@ -26,7 +26,10 @@ def do_keygen(args):
 
 def do_getpub(args):
     key = keys.load(args.key)
-    key.emit_c()
+    if args.rust:
```

Comment:
Instead of adding --rust, the args should gain a --type which defaults to 'c' and has choices=['c','rust'], just in case any more formats are needed later.